server/middleware/configureEstablishment.js

```diff
@@ -15,22 +13,27 @@ const configureEstablishment = () => (req, res, next) => {
       req.session.establishmentPersonalisationEnabled === undefined)
   ) {
     const replaceUrl = /-prisoner-content-hub.*$/g;
-    const establishmentName = pathOr('wayland', ['headers', 'host'], req)
+
+    const establishmentName = (req.headers?.host || 'wayland')
```

Comment:
Is a default required for this? Seemingly it masks a genuine issue?

Reply:
ah.. just said that above 👍 

We'll leave and fix in a follow up - I'd feel much more comfortable stripping out all the defaulting to wayland